platform-service-impl/src/main/java/esa/mo/platform/impl/provider/opssat/PowerControlOPSSATAdapter.java

```diff
@@ -43,20 +48,34 @@
 public class PowerControlOPSSATAdapter implements PowerControlAdapterInterface
 {
 
+  enum STATUS_MASK {
+	DEVICE_STATUS_SEPP1_MASK(0x0004, PayloadDevice.SEPP1),
+	DEVICE_STATUS_SEPP2_MASK(0x0008, PayloadDevice.SEPP2),
+	DEVICE_STATUS_SBAND_MASK(0x0020, PayloadDevice.SBandTRX),
+	DEVICE_STATUS_XBAND_MASK(0x0080, PayloadDevice.XBandTRX),
+	DEVICE_STATUS_SDR_MASK(0x0100, PayloadDevice.SDR),
+	DEVICE_STATUS_IADCS_MASK(0x0200, PayloadDevice.FineADCS),
+	DEVICE_STATUS_OPT_MASK(0x0400, PayloadDevice.OpticalRX),
+	DEVICE_STATUS_CAM_MASK(0x1000, PayloadDevice.HDCamera);
+
```

Comment:
Now that we track status in powerAdapter should we alos add GPS device here? Also in the init() method?

Reply:
Yes, absolutely!